ui/raidboss/timeline.js

```diff
@@ -787,29 +787,76 @@ class TimelineUI {
   }
 }
 
+const CountdownState = {
+  none: 0,
+  active: 1,
+  wipe: 2,
+};
+
 class TimelineController {
   constructor(options, ui) {
     this.options = options;
     this.ui = ui;
     this.dataFiles = {};
     // data files not sent yet.
     this.timelines = null;
+
+    // Fix for #1453
+    // Countdown state
+    // 0 = No countdown
+    // 1 = Countdown active
+    // 2 = Wiped during countdown
+    this.countdownState = CountdownState.none;
+    this.wipeRegex = Regexes.network6d({ command: '40000010' });
   }
 
   SetPopupTextInterface(popupText) {
     this.ui.SetPopupTextInterface(popupText);
   }
 
   SetInCombat(inCombat) {
+    // On wipe, set countdown state to Wipe if it was Active
+    if (!inCombat && this.countdownState === CountdownState.active)
+      this.countdownState = CountdownState.wipe;
```

Comment:
Does the `this.wipeRegex` check below already cover this block?

Reply:
This is to specifically handle this call to `StopCombat`, from `popup-text.js` line 310, which then invokes `SetInCombat(false)` on the timeline controller:
```javascript
  SetInCombat(inCombat) {
    if (this.inCombat == inCombat)
      return;

    // Stop timers when stopping combat to stop any active timers that
    // are delayed.  However, also reset when starting combat.
    // This prevents late attacks from affecting |data| which
    // throws off the next run, potentially.
    this.inCombat = inCombat;
    if (!this.inCombat) {
      this.StopTimers();
      this.timelineLoader.StopCombat();
    }
    if (this.inCombat)
      this.Reset();
  }
```

I'm not sure if this is an edge case that is likely to be seen, but I included it in the logic anyways to be thorough.

Reply:
Ah I see, because it stops processing logs when the timeline isn't running.  Got it.  :+1: